core/capabilities/outgoing_connector_handler.go

```diff
@@ -1,38 +1,56 @@
-package target
```

Comment:
@agparadiso This feels like the wrong place to me: can we put it in webapi instead?